Automattic/wp-calypso
# Add empty state to My Memberships tab.

#### Testing instructions
1. Make sure you're logged in to an account with no active membership plans
2. Visit https://wordpress.com/me/purchases/memberships

#### Question
Do we need both heading and a description? I'm considering going with "No memberships found" without the extra heading above.

Before:
![image](https://user-images.githubusercontent.com/4550351/58139861-0dc23400-7c80-11e9-9619-f3c2aca6ea66.png)


After:
![image](https://user-images.githubusercontent.com/4550351/58139846-ff741800-7c7f-11e9-869d-9df41adf3df9.png)


Fixes #32770
